packages/uxpin-merge-cli/bin/uxpin-merge

```diff
@@ -33,6 +33,7 @@ program
   .option('--cwd <path>', 'working directory: path to root of the DS repository')
   .option('--config <path>', 'path to a config file. Default: `./uxpin.config.js`')
   .option('--token <token>', 'auth token for the library. Default: `UXPIN_AUTH_TOKEN` env variable')
+  .option('--branch <branch>', 'branch to use when uploading.')
```

Comment:
Should we mention here that `master` is used if nothing is specified?
`.option('--branch <branch>', 'branch to use when uploading. Default: master')`

Reply:
Yup -- I'll add that note for sure, that will make it even more clear, I don't think we note that master is the default anywhere as of now.

- [ ] Add note to command line option description regarding branch default being `master`